packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.controller.js

```diff
@@ -24,29 +24,48 @@ export default class TelecomPackMigrationConfirmCtrl {
     };
 
     this.PROMO_DISPLAY = PROMO_DISPLAY;
+    this.QUANTITY = QUANTITY;
 
     this.modemTransportPrice = 9.99;
 
     this.process = this.TucPackMigrationProcess.getMigrationProcess();
     this.choosedAdditionalOptions = this.TucPackMigrationProcess.getOptionsSelected();
 
-    const modemRental = this.process.selectedOffer.modemRental
-      ? this.process.selectedOffer.modemRental.value
-      : 0;
+    let gtrComfortSelected = 0;
+    angular.forEach(this.choosedAdditionalOptions, (option) => {
```

Comment:
Suggestion: Can you use Array.prototype.forEach() here.

Reply:
done